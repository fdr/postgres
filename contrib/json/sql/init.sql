CREATE EXTENSION json;

\set ECHO none
SET client_min_messages = warning;

\i sql/test_strings.sql
CREATE TABLE valid_test_strings AS
	SELECT string FROM test_strings WHERE json_validate(string);

RESET client_min_messages;
\set ECHO all
