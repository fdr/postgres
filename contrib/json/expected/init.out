CREATE EXTENSION json;
\set ECHO none
